integration-cli/docker_cli_daemon_test.go

```diff
@@ -2332,3 +2332,51 @@ func (s *DockerDaemonSuite) TestBuildOnDisabledBridgeNetworkDaemon(c *check.C) {
 	c.Assert(err, check.IsNil, comment)
 	c.Assert(code, check.Equals, 0, comment)
 }
+
+// Test case for #22471
+func (s *DockerDaemonSuite) TestDaemonShutdownTimeout(c *check.C) {
+	testRequires(c, SameHostDaemon, DaemonIsLinux)
```

Comment:
@jhowardmsft Thanks for point that out. I will update the PR to cover Windows as well.